### DISCUSSION TITLE
Welcome to gerrit-code-review-plugin Discussions!

### DISCUSSION THREAD

**Discussion by lucamilanesio** [jenkinsci/gerrit-code-review-plugin#78] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by reda-alaoui** [comment]
Thank you @lucamilanesio :)

Hello from France!

I falled in love with Gerrit circa 2017. Since then, we have been using it everywhere in our company ( @Cosium ).
I made a few contributions to Gerrit core and the [its plugin](https://gerrit.googlesource.com/plugins/its-base/) .
I am currently the maintainer of https://github.com/jenkinsci/sonar-gerrit-plugin .

Since 2021, my company has been a heavy user of `gerrit-code-review-plugin`.

I want to help maintain the plugin as much as I can :)

**Comment by nickb-carallon** [comment]
Hi!

We're using Gerrit at our company @Carallon.

I'm current making changes to the Gerrit Code Review plugin to speed up the scanning of our repositories. To do this I'm attempting to remove all git checkouts done as part of the scan, similar to how other branch source plugins work (such as github-branch-source-plugin and gitlab-branch-source-plugin). This should take the time to scan from minutes to seconds.

Thanks for creating this plugin @lucamilanesio!

**Comment by ningboRainbow** [comment]
Thank you @lucamilanesio :)

Hello from China!
I installed the latest version of the gerrit-code-review-plugin and added the code-review function to our groovy script as you did in your example, but I found that it did not take effect and did not vote for 1 point even if the build was successful，pls help me？ths

![image](https://user-images.githubusercontent.com/92293880/221130598-1e4eb6f1-5115-4bbe-af85-af94a1a7af83.png)

![image](https://user-images.githubusercontent.com/92293880/221130464-4b245485-d107-4bea-9141-a33bde818c58.png)

![image](https://user-images.githubusercontent.com/92293880/221130867-043f6753-523d-4275-97f1-59b7d89032b5.png)

But it can vote 1 point by using the following form
![image](https://user-images.githubusercontent.com/92293880/221130957-8b990d65-2f78-4272-9e30-d6928f07d20e.png)

**Reply by ningboRainbow** [reply]
The problem of voting+1 has been solved, 
gerritReview labels: [Verified: 1], message: env.CODE_REVIEW_INFO   ===》
gerritReview labels: ["Code-Review": 1], message: env.CODE_REVIEW_INFO

but that message cannot be passed in through variables has not been solved
the form passed in according to the above variables does not take effect, so a default value "SUCCESS" is still given
can you help me？

**Reply by ningboRainbow** [reply]
errors like this:
Gerrit Review requires authentication, however there are no credentials defined or are empty.

**Comment by ningboRainbow** [comment]
Is this error related to the following variable ：GERRIT_CREDENTIALS_ID ? What should it be？

**Comment by felipecrs** [comment]
@lucamilanesio I cannot open any new discussion, I believe it's locked perhaps?

**Reply by bkhouri** [reply]
I'm unable to open new discussion too.  Is there another forum to reach out to the Gerrit Code Review Jenkins plugin community?

**Comment by chenhayat** [comment]
Hi,
I am using this plugin without using the trigger plugin (so I set the necessary env variable in the jenkins job).
I looked at your source code and wondered regarding the following 2 variables (related to the "gerritReview" functionality): 
1. GERRIT_PATCHSET_REVISION - I though that it could replace the need to pass branch name (or change number + patchset number), but it still requests this variable, and I don't see in source code that you are doing something with this.
Am I missing something ?  Is there  a way to pass an hash instead of change id and patchset number?
2. GERRIT_TOPIC - I set it as an env variable but message was still posted on gerrit review though the topic was different (I don't use the trigger plugin but thought it might publish it according to condition of topic), I looked at the source code and haven't seen that it is used as a condition except for setting it in the changed data. Have I missed anything here ?
Thanks :)
